src/staticsampler.jl

```diff
@@ -37,15 +37,15 @@ Static nested sampler with `nactive` active points and `ndims` parameters.
 
 `bounds` declares the Type of [`Bounds.AbstractBoundingSpace`](@ref) to use in the prior volume. The available bounds are described by [`Bounds`](@ref). `proposal` declares the algorithm used for proposing new points. The available proposals are described in [`Proposals`](@ref). If `proposal` is `:auto`, will choose the proposal based on `ndims`
 * `ndims < 10` - [`Proposals.Uniform`](@ref)
-* `ndims ≥ 10` - [`Proposals.RWalk`](@ref)
+* `10 <= ndims <= 20` - [`Proposals.RWalk`](@ref) or [`Proposals.RStagger`](@ref)
```

Comment:
I would remove this change-

Putting the upper bound of 20 doesn't make sense, because we're not providing anything different- if the number of dimensions is 23, we're still going to return `Proposals.RWalk()`. See the code here https://github.com/TuringLang/NestedSamplers.jl/blob/c3057f8b2fb82a4dd90aaa2a9255280234dcd161/src/staticsampler.jl#L63-L70 which is what this docstring describes.

Reply:
yes, sounds reasonable, until more features are added shouldn't be including the upper bound of 20